### ISSUE TITLE
Request: Pull Inklecate.js into its own package

### ISSUE THREAD

**Issue by sequitur** [inkle/inky#96]
Is the inklecate.js compiler that Inky uses written just for Inky itself? Would it be possible then to pull that out into its own NPM module? It would make it easier to build js-based development tools for ink (like, say, Atom packages with Inky-like functionality) while keeping track of changes to Ink.

**Comment by joethephish** [comment]
Yeah it's two things - a wrapper around the command line binary tool inklecate (part of ink), as well as dealing with the main process side of the IPC messaging within electron, so needs a load of management for compilation/play requests coming in somewhat asynchronously.

It would be possible to pull it out into a separate module a guess, though I'm afraid wouldn't be something that we'd have the time to do. If fancied giving it a try, if warn you that that code is slightly hacky!

**Comment by Julisep** [comment]
[**Here**](https://github.com/Julisep/Inkjs-Compiler) it is.

Unfortunately I do not have the time right now (A-Levels) to document the usage, but it supports all features supported in Inky.
